operator/grafana/generate-configmap.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+for file in *.json; do
```

Comment:
this is not necessary, instead use the kustomize configMapGenerator https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#configmapgenerator